contracts/core/RebalancingSetToken.sol

```diff
@@ -178,10 +185,18 @@ contract RebalancingSetToken is
         require(block.timestamp >= lastRebalanceTimestamp.add(rebalanceInterval));
 
         // Check that new proposed Set is valid Set created by Core
-        require(ICore(ISetFactory(factory).core()).validSets(_rebalancingSet));
+        require(ICore(ISetFactory(factory).core()).validSets(_nextSet));
+
+        // Check that the propoosed set is a multiple of current set, or vice versa
+        uint256 currentNaturalUnit = ISetToken(currentSet).naturalUnit();
+        uint256 rebalancingNaturalUnit = ISetToken(_nextSet).naturalUnit();
+        require(
+            Math.max256(currentNaturalUnit, rebalancingNaturalUnit) %
```

Comment:
If we're going with `nextSet`, then the name of `rebalancingNaturalUnit` should be `nextSetNaturalUnit` or `nextNaturalUnit`

Reply:
Ah yes I agree...missed that one